operations/patch_util.go

```diff
@@ -451,6 +475,29 @@ func getPatchDisplay(p *patch.Patch, summarize bool, uiHost string) (string, err
 	return out.String(), nil
 }
 
+// getCommitQueuePatchDisplay returns a human-readable summary representation of a patch object
+// which can be written to the terminal, and links to the new UI commit queue page for
+// the project in question in the CLI
+func getCommitQueuePatchDisplay(p *patch.Patch, summarize bool, uiHost string) (string, error) {
```

Comment:
Rather than having another function for this, getPatchDisplay can just use `p.GetCommitQueueURL` if `IsCommitQueuePatch` is true.